### PR TITLE
Update bedrock-entry.sh

### PR DESCRIPTION
**Pull request by vcaboara**
Change `16` to `16.1` and added `16.2`.

### CODE REVIEW

**Comment by double16** [comment]
Shouldn’t 16.2 be 16.20?

**Comment by itzg** [comment]
> Shouldn’t 16.2 be 16.20?

Odd, Bedrock is doing some kind of non-standard jump-by-tens thing with their versions

image

So much for lining up with the Java edition versions.

**Comment by vcaboara** [comment]
We have Minecraft on the Nintendo Switch, and it ONLY plays with matching versions. :(
I'd be happy to update `2` -> `20` if desired.

Thanks for your time and work @itzg!

File: bedrock-entry.sh [hunk 1]
```diff
@@ -27,9 +27,12 @@ case ${VERSION^^} in
   1.14|PREVIOUS)
     VERSION=1.14.60.5
     ;;
-  1.16)
+  1.16.1)
     VERSION=1.16.1.02
     ;;
+  1.16.2)
```
Comment (vcaboara) [inline bedrock-entry.sh:33]:
```suggestion
  1.16.20)
```

File: bedrock-entry.sh [hunk 2]
```diff
@@ -27,9 +27,12 @@ case ${VERSION^^} in
   1.14|PREVIOUS)
     VERSION=1.14.60.5
     ;;
-  1.16)
+  1.16.1)
     VERSION=1.16.1.02
     ;;
+  1.16.20)
```
Comment (itzg) [inline bedrock-entry.sh:33]: I think if you change this to simply "1.16" then that'll be a decent compromise for the different clients out there.

Comment (vcaboara) [inline bedrock-entry.sh:33]:
```suggestion
  1.16)
```

**Pull request merged**